core/workspace.js

```diff
@@ -770,25 +783,27 @@ Blockly.Workspace.prototype.setVariableMap = function(variableMap) {
  * Database of all workspaces.
  * @private
  */
-Blockly.Workspace.WorkspaceDB_ = Object.create(null);
+Workspace.WorkspaceDB_ = Object.create(null);
```

Comment:
Since this is both private and static, I think you can just do `const workspaceDB = Object.create(null);` and remove the private annotation/replace uses of it in this file.